<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMZB</class>
 <widget class="QWidget" name="FormMZB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1328</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mekton Zéta Builder v0.0.0.0.1</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>1311</width>
     <height>691</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabCrea">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabChassi">
       <attribute name="title">
        <string>Chassis</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1281</width>
          <height>621</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayoutChassis" rowstretch="0,0,5" columnstretch="0,1,1,1,1,1,1,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item row="0" column="6">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Points de Blindage</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Type Blindage</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Retirer le Servo</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labelType">
           <property name="text">
            <string>Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string>Nom du servo</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonAjoutServo">
           <property name="text">
            <string>Ajouter un servo</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelLocalisation">
           <property name="text">
            <string>Localisation</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="spacerChassis">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Espace</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Kills</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="labelPoidsChassis">
           <property name="text">
            <string>Poids Chassis : N/A</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QLabel" name="labelPCChassis">
           <property name="text">
            <string>PC Chassis : N/A</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabSAOpt">
       <attribute name="title">
        <string>Sous Assemblage et Options</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabArmes">
       <attribute name="title">
        <string>Armes</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabProp">
       <attribute name="title">
        <string>Propulsion</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Récapitulatif</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelMVSol">
        <property name="text">
         <string>MV au Sol : N/A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMVVol">
        <property name="text">
         <string>MV en Vol : N/A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelVM">
        <property name="text">
         <string>VM : N/A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPoids">
        <property name="text">
         <string>Poids : N/A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPC">
        <property name="text">
         <string>PC : N/A</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
